### DISCUSSION TITLE
Automatic download of mods required for a mod to work.

### DISCUSSION THREAD

**Discussion by ghost** [PrismLauncher/PrismLauncher#1531] [Ideas]
For example, let's say that adastra mod requires botarium mod to be installed, it would be nice if it could be automatically installed in botarium mod when I install adastra. Curseforge's launcher can do this. Modrinth launcher can also do it. Adding such a feature can be very useful to modpack makers.

**Comment by Trial97** [comment]
Already implemented in the nightly builds:https://github.com/PrismLauncher/PrismLauncher/pull/986
It will be released in 8.0

**Reply by ghost** [reply]
I'm sorry, I didn't notice, thank you.
